clients/python/.gitlab-ci.yml

```diff
@@ -0,0 +1,24 @@
+# ref: https://docs.gitlab.com/ee/ci/README.html
```

Comment:
Can you add a comment about this file?
I have 0 understanding of why we need it.

Reply:
I'll remove it and add it to the ignore list so the generator will not re-create it.